Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'rr', '~> 1.1.2'
 gem 'slop', '~> 3.6.0'
 gem 'rspec', '~> 3.2.0'
 gem 'rake'
-gem 'wiktionary'
+gem 'wiktionary', :path => '../wiktionary/'
```

Comment:
I am wondering about this change. Since this goes to master, we should avoid using unspecified version of the dependencies. I think that a better solution is a `git:` key with pointer to a github repo (like in the case of progress).